### ISSUE TITLE
AWSManagedDomainsAggregateThreatList throws error

### ISSUE THREAD

**Issue by farmerbean** [awslabs/landing-zone-accelerator-on-aws#101]
**Describe the bug**
When including AWSManagedDomainsAggregateThreatList in the Route53 firewall RuleGroups, LZA throws an error in the buildstage:

**To Reproduce**
Include AWSManagedDomainsAggregateThreatList as a firewallRuleGroup:
```
* Value should be one of "AWSManagedDomainsBotnetCommandandControl",
* "AWSManagedDomainsMalwareDomainList"
* at /centralNetworkServices/route53Resolver/firewallRuleGroups/0/rules/1/managedDomainList
* in iam-config.yaml config file
* 
```

**Expected behavior**
no error

**Please complete the following information about the solution:**

- [1.3.2 ] Version: [e.g. v1.1.0]

- [euw1 ] Region: [e.g. us-east-1]

**Comment by erwaxler** [comment]
Hi @farmerbean , thank you for submitting this issue! This feature is under development and will be included in the next release. I will update this issue once this has been implemented.

**Comment by awsclemj** [comment]
Hello, 

This value is now available as of our v1.5.0 release. Please see the [documentation](https://awslabs.github.io/landing-zone-accelerator-on-aws/classes/_aws_accelerator_config.DnsFirewallRulesConfig.html#managedDomainList) for more details. Thank you for your feedback and support!
